python/lib/dependabot/python/update_checker.rb

```diff
@@ -283,6 +280,26 @@ def poetry_library?
         false
       end
 
+      def updating_pipfile?
+        requirement_files.any?("Pipfile")
+      end
+
+      def updating_pyproject?
```

Comment:
Would it be better/more clear if we used `poetry.lock` for this determination and renamed this method `updating_poetry`? Since `pyproject.toml` is used by other package managers like flit and hatch I think tying poetry to pyproject.toml may be misleading.

Reply:
Right, I think this will make more sense with #5661, where this logic will apply to all ways of updating `pyproject.toml`. For now it doesn't make a difference because Poetry is the only supported way of updating a `pyproject.toml` file. In conclusion, I'm happy to change it but I'll probably change it back in #5661 so probably not worth it?